pkg/storage/tpr/fakes/clientset_1_5.go

```diff
@@ -0,0 +1,1284 @@
+/*
+* Copyright 2016 The Kubernetes Authors.
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*     http://www.apache.org/licenses/LICENSE-2.0
+*     Unless required by applicable law or agreed to in writing, software
+*     distributed under the License is distributed on an "AS IS" BASIS,
+*     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*     See the License for the specific language governing permissions and
+*     limitations under the License.
+ */
+
+// This file was generated by counterfeiter
```

Comment:
It is a fake(mock) generator. I learnt about it during my work in Cloud Foundry
https://github.com/maxbrunsfeld/counterfeiter (now included in PR description)